compiler/src/dmd/globals.d

```diff
@@ -173,6 +148,28 @@ extern (C++) struct Param
     bool hcUsage;           // print help on -HC switch
     bool logo;              // print compiler logo
 
+    // Options for `-preview=/-revert=`
+    FeatureState useDIP25;       // implement https://wiki.dlang.org/DIP25
+    FeatureState useDIP1000;     // implement https://dlang.org/spec/memory-safe-d.html#scope-return-params
+    bool ehnogc;                 // use @nogc exception handling
+    bool useDIP1021;             // implement https://github.com/dlang/DIPs/blob/master/DIPs/accepted/DIP1021.md
```

Comment:
I feel these `bool`s should all be `FeatureState`s, but I'll leave this to a separate PR.